README.md

```diff
@@ -1 +1,46 @@
-# bff-samples
\ No newline at end of file
+## Ballerina Persists module with databases
+
+Ballerina's persistence features offer a straightforward way to work with data sources, making it easier to build an application's data access layer. By simply defining the required Ballerina records, the persistence package auto-generates all data access functions. This eliminates repetitive code, providing a simplified interface for inserting, updating and querying data with data tables directly mapped to Ballerina records.
```

Comment:
Shouldn't we reword `Ballerina's persistence features` to `The bal persist feature`? We refer to it as `bal persist` feature. So better be consistent

@daneshk wdyt?

Reply:
+1

Reply:
`The bal persist feature` in here, shall we add a hyperlink to the word `persist` that redirect to the `persist` API docs. wdyt?

Reply:
Added.